### PR TITLE
Add F# to dotnet cli 1..1.0

### PR DESCRIPTION
**Pull request by KevinRansom**
Add F# support into the dotnet sdk and dotnet cli 1.1.0

### CODE REVIEW

**Comment by KevinRansom** [comment]
@livarcocc @nguerrera @srivatsn  @cartermp @brettfo 

Hey guys want to take a look at this please.

File: Microsoft.DotNet.Cli.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
```
Comment (livarcocc) [inline Microsoft.DotNet.Cli.sln]: Somewhat concerned about all the changes that happened on this file. Don't really understand why they happened.

File: build/Microsoft.DotNet.Cli.Compile.targets [hunk 2]
```diff
@@ -194,7 +201,8 @@
                 JITPath="$(LibCLRJitPath)"
                 CrossgenPath="$(CrossgenPath)"
                 ReadyToRun="True"
-                PlatformAssemblyPaths="@(PlatformAssemblies);
+                PlatformAssemblyPaths="%(CrossgenTargets.RootDir)%(CrossgenTargets.Directory);
```
Comment (livarcocc) [inline build/Microsoft.DotNet.Cli.Compile.targets]: Why do we need this?

File: src/tool_fsharp/tool_fsc.csproj [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+          AfterTargets="ComputeFilesToPublish">
+    <ItemGroup>
+      <!-- Workaround for https://github.com/dotnet/sdk/issues/115 -->
+      <HackFilesToCopy Include="$(NuGetPackagesDir)\Microsoft.FSharp.Compiler\$(CLI_FSharp_Version)\contentFiles\any\any\**;" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(HackFilesToCopy)"
+        DestinationFiles="@(HackFilesToCopy->'$(PublishDir)/%(RecursiveDir)%(Filename)%(Extension)')" />
+  </Target>
+  <Target Name="MakeFscRunnableAndMoveToPublishDir"
```
Comment (livarcocc) [inline src/tool_fsharp/tool_fsc.csproj]: NIT: empty space here.

**Comment by livarcocc** [comment]
No tests for this new way of doing F#?

**Comment by livarcocc** [comment]
@KevinRansom This should not be targeting master. This needs to target rel/1.1.0.

**Comment by KevinRansom** [comment]
@I will fix the targeting, no tests possible until the templates are updated.  Then the existing tests should continue to work fine.

**Comment by KevinRansom** [comment]
need to retarget.

**Pull request closed without merging**